docs/INSTALL.md

```diff
@@ -119,20 +118,40 @@ To configure/install the UPF in DPDK mode, the following changes are required:
      # In the order of (n-s1u n-sgi)
     -nhmacaddrs=(22:53:7a:15:58:50 22:53:7a:15:58:50)
     +nhmacaddrs=(b4:96:91:b4:47:b8 b4:96:91:b4:47:b9)
-
```

Comment:
Why does this empty line need to be removed? It is part of the output of git diff

Reply:
ok